manpages/fsck.fat.8.in

```diff
@@ -103,125 +103,129 @@ The other is the legacy Atari variant used on Atari ST.
 By default, unused clusters are added to the free disk space except in auto mode
-(\fB\-a\fR).
-.IP "\fB\-l\fR" 4
+(\fB\-a\fP).
+.IP "\fB\-l\fP" 4
 List path names of files being processed.
-.IP "\fB\-n\fR" 4
-No\-operation mode: non\-interactively check for errors, but don't write
+.IP "\fB\-n\fP" 4
+No-operation mode: non-interactively check for errors, but don't write
 anything to the filesystem.
-.IP "\fB\-p\fR" 4
-Same as \fB\-a\fR, for compatibility with other *fsck.
-.IP "\fB\-r\fR" 4
+.IP "\fB\-p\fP" 4
+Same as \fB\-a\fP, for compatibility with other *fsck.
+.IP "\fB\-r\fP" 4
 Interactively repair the filesystem.
 The user is asked for advice whenever there is more than one approach to fix an
 inconsistency.
 This is the default mode and the option is only retained for backwards
 compatibility.
-.IP "\fB\-S\fR" 4
+.IP "\fB\-S\fP" 4
 Consider short (8.3) file names with spaces in the middle to be invalid, like
 previous versions of this program did.
 While such file names are not forbidden by the FAT specification, and were never
 treated as errors by Microsoft file system checking tools, many DOS programs are
 unable to handle files with such names. Using this option can make them
 accessible to these programs.
 .IP "" 4
-Short file names which \fIstart\fR with a space are considered invalid
+Short file names which \fIstart\fP with a space are considered invalid
 regardless of this option's setting.
 .IP "" 4
-Previous versions of this program exceptionally treated "EA DATA . SF" and
-"WP ROOT . SF" as valid short names; using this option does not preserve that
-exception.
-.IP "\fB\-t\fR" 4
+Previous versions of this program exceptionally treated \fIEA DATA . SF\fP and
+\fIWP ROOT . SF\fP as valid short names; using this option does not preserve
+that exception.
+.IP "\fB\-t\fP" 4
 Mark unreadable clusters as bad.
-.IP "\fB\-u\fR \fIPATH\fR" 4
+.IP "\fB\-u\fP \fIPATH\fP" 4
 Try to undelete the specified file.
-\fBfsck.fat\fR tries to allocate a chain of contiguous unallocated clusters
+\fBfsck.fat\fP tries to allocate a chain of contiguous unallocated clusters
 beginning with the start cluster of the undeleted file.
 This option can be given more than once.
-.IP "\fB\-v\fR" 4
+.IP "\fB\-v\fP" 4
 Verbose mode.
 Generates slightly more output.
-.IP "\fB\-V\fR" 4
+.IP "\fB\-V\fP" 4
 Perform a verification pass.
 The filesystem check is repeated after the first run.
 The second pass should never report any fixable errors.
 It may take considerably longer than the first pass, because the first pass may
 have generated long list of modifications that have to be scanned for each disk
 read.
-.IP "\fB\-\-variant\fR \fITYPE\fR" 4
-Create a filesystem of variant \fITYPE\fR.
-Acceptable values are 'standard' and 'atari' (in any combination of upper/lower
-case).
+.IP "\fB\-\-variant\fP \fITYPE\fP" 4
+Create a filesystem of variant \fITYPE\fP.
+Acceptable values are \fIstandard\fP and \fIatari\fP (in any combination of
+upper/lower case).
```

Comment:
Absolutely not.  The quotes are acceptable (though double quotes would be better).  If you consider "standard" and "atari" as keywords, then make it

    Acceptable values are
    .B standard
    and
    .B atari
    (in any combination of ...

Reply:
GNU man -Tutf8 renders 0x27 (') character as Unicode Right Single Quotation Mark. So quoting word with single quotes (') leads to two right quotation marks.